data-prepper-plugins/opensearch/src/main/java/com/amazon/dataprepper/plugins/sink/opensearch/index/IsmPolicyManagement.java

```diff
@@ -27,36 +33,60 @@
     // TODO: replace with new _opensearch API
     private static final String POLICY_MANAGEMENT_ENDPOINT = "/_opendistro/_ism/policies/";
     public static final String DEFAULT_INDEX_SUFFIX = "-000001";
+    private static final String POLICY_FILE_ROOT_KEY = "policy";
+    private static final String POLICY_FILE_ISM_TEMPLATE_KEY = "ism_template";
 
     private final RestHighLevelClient restHighLevelClient;
     private final String policyName;
-    private final String policyFileWithIsmTemplate;
+    private final String policyFile;
     private final String policyFileWithoutIsmTemplate;
 
     public IsmPolicyManagement(final RestHighLevelClient restHighLevelClient,
```

Comment:
Nit: if this constructor is only invoked by IndexManagerFactory for trace-analytics, we should change the modifier to protected

Reply:
George, Thanks a lot for the review! Good point, we should be able to change it from public to package private. Will make this change in an upcoming pull request.